platform/upgrades/adding-custom-cloud-credentials.md

```diff
@@ -304,6 +316,72 @@ Grid attempts to delete all cluster resources when a delete operation is initiat
 grid delete cluster <cluster name>
 ```
 
+## Cost saving mode
+
+There are two cluster management modes you can pick, depending on your expected cluster size and latency/cost preferences.
```

Comment:
```suggestion
To save on your cluster management cost, you can activate the `--cost-savings` flag when creating your cluster.

Turning this flag to true will reduce some of the cost, but can increase start-up times for experiment, tensorboard logs and datastores. This will not impact sessions start times or runtime speed.

The actual cost is dependant on the region, but it can reduce to cost to ~10$ a day, compared to ~50$ using the default mode.
```